pkg/kubelet/apis/config/helpers_test.go

```diff
@@ -132,6 +132,7 @@ var (
 	kubeletConfigurationPathFieldPaths = sets.NewString(
 		"StaticPodPath",
 		"Authentication.X509.ClientCAFile",
+		"CertDirectory",
```

Comment:
please see comment for this collection: 

> `// KubeletConfiguration fields that contain file paths. If you update this, also update KubeletConfigurationPathRefs!`

Reply:
I referred to this (https://github.com/kubernetes/kubernetes/pull/88480/files)  and missed it, I will add it. I will also try and make these tests dynamic. 

https://github.com/kubernetes/kubernetes/blob/617064d7327bf455955cb785d57330c425d48b51/pkg/kubelet/apis/config/helpers_test.go#L132-L138 in the same PR before the next rev.